pom.xml

```diff
@@ -39,7 +39,7 @@
             <dependency>
                 <groupId>uk.gov.justice</groupId>
                 <artifactId>maven-common-bom</artifactId>
-                <version>1.15.0</version>
+                <version>1.28.0</version>
```

Comment:
Do we typically have these as properties?

Reply:
Sometimes although arguably the BOM version is likely only used the once.
I have since updated this